RGO.Services.Tests/Services/EmployeeServiceUnitTests.cs

```diff
@@ -463,8 +463,16 @@ public async Task GetCurrentMonthTotalReturnsExistingTotalTest()
 
         var currentYear = DateTime.Now.Year;
 
-        MonthlyEmployeeTotalDto monthlyEmployeeTotalDto = new(1, 1, 1, 0, 0, 0, currentMonth, currentYear);
-
+        MonthlyEmployeeTotalDto monthlyEmployeeTotalDto = new MonthlyEmployeeTotalDto
+        {
+            Id = 1,
+            EmployeeTotal = 1,
+            DeveloperTotal = 1,
+            ScrumMasterTotal = 0,
+            BusinessSupportTotal = 0,
+            Month = currentMonth,
+            Year = currentYear
+        };
```

Comment:
Can we add this to the TestData as the same data is used and created 7 times in this file

Reply:
Added as test data